extensions/amp-story/1.0/amp-story.js

```diff
@@ -444,14 +444,19 @@ export class AmpStory extends AMP.BaseElement {
       return;
     }
     // The theme color should be copied from the story's primary accent color
-    // if possible, with the fall back being default light gray.
+    // if possible, with the fall back being default dark gray.
     const meta = this.win.document.createElement('meta');
     const ampStoryEl = this.win.document.getElementsByTagName('amp-story')[0];
-    const styles = computedStyle(this.win, ampStoryEl);
+    const ampStoryPageEl =
```

Comment:
nit: I think you can do `ampStoryEl.querySelector('amp-story-page')[0]` and it will be somewhat of a performance gain, since you'll only look for elements that are children of `amp-story` instead of anywhere else in the document.